passes/techmap/abc.cc

```diff
@@ -655,8 +655,9 @@ struct abc_output_filter
 };
 
 void abc_module(RTLIL::Design *design, RTLIL::Module *current_module, std::string script_file, std::string exe_file,
```

Comment:
it's not specifically for this PR but I can't help but think a struct of configuration options would be neater than this monstrous set of function arguments at this point

Reply:
yes, that's a mood. each time I add something to it I feel that  :D